apps/site/src/views/Resources/sections/ApiResources/ApiResources.tsx

```diff
@@ -1,27 +1,26 @@
 import { PortableText } from "@portabletext/react";
+import { client } from "@/lib/sanity/client";
```

Comment:
Chore: client is defined in our own code, so where it was initially placed was correct.